src/wsi/sdl2/wsi_platform_sdl2.h

```diff
@@ -2,20 +2,107 @@
 
 #include <SDL2/SDL.h>
```

Comment:
Similarly this is canonically

```suggestion
#include "SDL.h"
```

in SDL2.